docs/output_files.md

```diff
@@ -0,0 +1,71 @@
+
+## Output files
+
+*Nomadic* produces several files that provide information about the quality of your sequencing run, and the variants that were detected. They can be found in the results directory (`results/<expt_name>`), and are described briefly below:
+
+### `summary.bam_flagstats.csv`
+The `summary.bam_flagstats.csv` contains information about read mapping for each sample.
+
+Each row corresponds to a sample, which can be identified by its barcode (e.g. `barcode01`). The values provided come from [`samtools flagstats`](https://www.htslib.org/doc/samtools-flagstat.html) and are counts of read mapping flags inside of the sample's BAM file. Together they summarise how well your sequencing reads mapped to your reference genome.
```

Comment:
WIth the open sample id PR this will also include sample id, maybe we want to mention it here

Reply:
Added.